Automattic/wp-calypso
# Marketplace: mobile category menu

#### Changes proposed in this Pull Request

* Adds mobile only handling for the category selector to replace dropdown with a swipe / scroll selector in its place

#### Testing instructions

* On mobile breakpoints < 660px, you should see the swipe-friendly category selector and be able to change category.
* At non mobile breakpoints the new selector from [this parent branch / pr](https://github.com/Automattic/wp-calypso/pull/63592) should render in its place.

https://user-images.githubusercontent.com/811776/169184383-4efcb2ee-a31a-4e5c-a429-e1c0c1969587.mp4

Related to https://github.com/Automattic/wp-calypso/issues/62891
